src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  let result = {};
+
+  for (const key in robot) {
+    if (robot[key] in result) {
```

Comment:
as a suggestion, you can use `hasOwnProperty`
```suggestion
    if (result.hasOwnProperty(robot[key])) {
```